glean-core/src/lib.rs

```diff
@@ -95,11 +100,13 @@ impl Glean {
         let mut glean = Self {
             upload_enabled,
             data_store: Database::new(data_path)?,
+            event_data_store: EventDatabase::new(data_path)?,
             core_metrics: CoreMetrics::new(),
             data_path: PathBuf::from(data_path),
             application_id,
             ping_registry: HashMap::new(),
             start_time: local_now_with_offset(),
+            max_events: DEFAULT_MAX_EVENTS,
```

Comment:
Are there any plans to make this configurable? If not, should we just keep it as a constant and not a config option?

Reply:
Yes, we need this for feature parity with glean-ac for now, [see here](https://github.com/mozilla-mobile/android-components/blob/8ba1a8d0ca4521fc95a371f308ffcf1075ca2ce1/components/service/glean/src/main/java/mozilla/components/service/glean/config/Configuration.kt#L51). We might revisit this decision in the future, though, as no product is using it right now.

Reply:
Ah -- I had meant to come back around to this.  I had initially made this a parameter to the Glean constructor, but that isn't going to scale well to the ~dozen config parameters we'll probably end of having, and would require updating calls to the Glean constructor in *all* of the tests.

A better pattern might be to have an explicit `Configuration` struct that's passed into the `Glean` constructor (as glean-ac does).  Ideas and brainstorming [in this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1564505), but I'd advocate doing that as a follow-on to this PR.

Reply:
yup, let's do a followup.